README.md

```diff
@@ -23,3 +23,64 @@ Links:
 - How to Contribute: https://docs.tsuru.io/stable/contributing/
 - Repository & Issue Tracker: https://github.com/tsuru/tsuru
 - Talk to us on Gitter: https://gitter.im/tsuru/tsuru
+
+## Quick Start 
+
+With the purpose of test tsuru and/or for development, you can use [installer](https://docs.tsuru.io/master/experimental/installer.html) to have tsuru up and running. This is an experimental feature
+so **DON'T USE FOR PRODUCTION!**
```

Comment:
I'd remove the use for production. Just say it's a experimental feature and let people decide whether they want to go to production with experimental stuff.